components/codec/src/stream_event.rs

```diff
@@ -71,6 +71,28 @@ impl Iterator for EventIterator {
     }
 }
 
+// TODO: It's added to solve the dependency problem. Remove it in the future
+// since there is a same implementation in tikv_util.
+#[derive(Debug, Clone, Copy)]
+pub enum Either<L, R> {
```

Comment:
I don't see the point of using `Either` here. The returned type of `encode_event` is `[impl AsRef<[u8]>; 4]`, which hide the concrete type. So technically, it's the same as just using `[[u8; 4]; 4]`.

Reply:
Removed, PTAL